src/functions-and-arrays.js

```diff
@@ -1,119 +1,197 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
-
-
+function maxOfTwoNumbers(num1, num2) {
+  if (num1 > num2) {
+    return num1;
+  } else {
+    return num2;
+  }
+}
+console.log(maxOfTwoNumbers(10, 9));
 
 // Iteration #2: Find longest word
-const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
+const words = [
+  "mystery",
+  "brother",
+  "aviator",
+  "crocodile",
+  "pearl",
+  "orchard",
+  "crackpot",
+];
 
-function findLongestWord() {}
+function findLongestWord(arr) {
+  if (!arr) return null;
```

Comment:
An `array` even empty it is equal to `true` so this function will never return `null` the only thing you need to do here to correct it is to check for the `length` of the `array` instead of the `array` it self:

`if (!arr.length) return null;`